### DISCUSSION TITLE
Translation guidelines

### DISCUSSION THREAD

**Discussion by niklashaug** [chatwoot/chatwoot#2062] [General]
Hi! I'm providing some German translations at the moment via Crowdin and I'm wondering whether there are guidelines available regarding translations. For instance in German there are two forms of "you" where one is rather used in formal contexts ("Sie") and the other one is used among friends ("Du"). It's pretty important to know which form to use as at the moment both forms are used without any consistency.

**Comment by pranavrajs** [comment]
@niklashaug Thanks for bringing this up. We should add this to the translation guidelines. At the moment, there is no written document for the context. We prefer to have formal language instead of informal. This discussion came up earlier in one of the PR. 

https://github.com/chatwoot/chatwoot/pull/1342#discussion_r506134063

I will add this to the translation guide.

**Reply by niklashaug** [reply]
@pranavrajs alright. Is there an option to mark translations as wrong and to provide corrections for them? How can I overwrite translations that have been done already? I could refactor existing translations if needed.

**Reply by pranavrajs** [reply]
@niklashaug I think you need to upvote the translation, let me see if there is another way to do it.

**Reply by pranavrajs** [reply]
Also, added this to the translation guide. https://github.com/chatwoot/docs/blob/main/docs/contributing-guide/translation-guidelines.md#formal-vs-informal-context

**Reply by niklashaug** [reply]
> @niklashaug I think you need to upvote the translation, let me see if there is another way to do it.

So is it enough to provide a correction and upvote the correction so that in the next releases the corrections will be displayed?

**Reply by pranavrajs** [reply]
I think that should be enough, let me try to bring in the translation and see if that works.

**Comment by miiton** [comment]
Hello,

I was reading the documentation to improve the Japanese translation and checked translate.chatwoot.com, but I couldn't find the corresponding section. The source code and translate.chatwoot.com are linked, correct?

Corresponding section: https://github.com/chatwoot/chatwoot/blob/master/config/locales/en.yml#L167-L170

I've only reviewed config/locales/en.yml, but if I submit a correction request on translate.chatwoot.com, will it still be reflected in future commits as per the current workflow?

**Reply by miiton** [reply]
I apologize, it appeared after changing the Workflow to All Content.

![image](https://github.com/user-attachments/assets/2698dab5-0c63-426e-a3fb-7cf71bec9435)
